### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by rachel-chann**
Please review this work @Connor-Whiteside - check naming, function doctring, etc.

### CODE REVIEW

File: prac_05/hex_colours.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"""
+CP1404/CP5632 Practical
+Hexadecimal colour codes in a dictionary
+"""
+
+NAME_TO_CODE = {"Absolute Zero": "#0048ba", "Baby Blue": "#89cff0", "Cadet Grey": "	#91a3b0", "Daffodil": "#ffff31",
```
Comment (Connor-Whiteside) [inline prac_05/hex_colours.py:6]: Good name

File: prac_05/hex_colours.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                "Earth Yellow": "#e1a95f", "Fawn": "#e5aa70", "GO Green": "#00ab66", "Han Blue": "#446ccf",
```
Comment (Connor-Whiteside) [inline prac_05/hex_colours.py:7]: GO Green will never occur due to the .title()

File: prac_05/word_occurrences.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+"""
+
+text = input('Text: ')
+words = text.split(' ')
+word_to_count = {}
+for word in words:
+    occurrence = word_to_count.get(word, 0)
+    word_to_count[word] = occurrence + 1
+max_length = max((len(word) for word in words))
+for word, count in sorted(word_to_count.items()):
```
Comment (Connor-Whiteside) [inline prac_05/word_occurrences.py:15]: This is a cool way of sorting out the words. Another possible method is changing word_to_count to a list then sorting it that way but your way honestly seems easier

File: prac_05/emails.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+"""
+
+
+def main():
+    """Create name_to_email dictionary."""
+    name_to_email = {}
+    email = input("Email: ")
+    while email != "":
+        name = get_name(email)
+        name_confirmation = input(f"Is your name {name}? (Y/n) ").upper()
```
Comment (Connor-Whiteside) [inline prac_05/emails.py:15]: Excellent naming

File: prac_05/wimbledon.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    countries = set()
+    for record in records:
+        countries.add(record[COUNTRY_INDEX])
+        champion_to_count[record[CHAMPION_INDEX]] = champion_to_count.get(record[CHAMPION_INDEX], 0) + 1
+    return champion_to_count, countries
+
+
+def print_results(champion_to_count, countries):
+    """Print champions and countries"""
+    print("Wimbledon Champions: ")
```
Comment (Connor-Whiteside) [inline prac_05/wimbledon.py:31]: Awesome naming throughout. Clear and concise.

**Review by Connor-Whiteside — COMMENTED** [review]
Overall, excellent variable names, and it was good to see some different methods of doing some of the task. Awesome work :)

**Comment by rachel-chann** [comment]
Thank you very much for reviewing my code @Connor-Whiteside

**Pull request merged**